src/common/wigmix-core.ts

```diff
@@ -471,6 +472,57 @@ export interface ColoredInterface {
   backgroundColorDark: string;
 }
 
+/**
+ * An interface for a trending product (will be changed when API is integrated)
```

Comment:
If we expect to introduce backward-incompatible change in a short time period, mark this as `@internal` first